CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+### Fixed
+- Issue [#76](https://github.com/reportportal/agent-js-playwright/issues/76) nested steps are not finished when `nestedSteps : true`
```

Comment:
Can we also mention [this issue](https://github.com/reportportal/agent-js-playwright/issues/97) as fixed?